pkg/metrics/metrics.go

```diff
@@ -22,30 +22,43 @@ import (
 )
 
 const (
-	metricGardenProjectsSum = "garden_projects_total"
-	metricGardenUsersSum    = "garden_users_total"
+	metricGardenProjectsSum    = "garden_projects_total"
+	metricGardenSeedsSum       = "garden_seeds_total"
```

Comment:
`count(garden_seed_info)` does the same thing as `garden_projects_total`. We try not to add unnecessary metrics. Please remove unless you have another reason for this metric.

Reply:
done